it/connectors/StateBenefitsConnectorISpec.scala

```diff
@@ -0,0 +1,341 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import com.github.tomakehurst.wiremock.http.HttpHeader
+import config.{AppConfig, BackendAppConfig}
+import connectors.StateBenefitsConnectorISpec.expectedResponseBody
+import helpers.WiremockSpec
+import models._
+import play.api.Configuration
+import play.api.http.Status._
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.{HeaderCarrier, HeaderNames, HttpClient, SessionId}
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import utils.DESTaxYearHelper.desTaxYearConverter
+
+class StateBenefitsConnectorISpec extends WiremockSpec {
+
+  lazy val connector: StateBenefitsConnector = app.injector.instanceOf[StateBenefitsConnector]
+  lazy val httpClient: HttpClient = app.injector.instanceOf[HttpClient]
+
+
+  def appConfig(desHost: String): AppConfig = new BackendAppConfig(app.injector.instanceOf[Configuration], app.injector.instanceOf[ServicesConfig]) {
+    override val desBaseUrl: String = s"http://$desHost:$wireMockPort"
+  }
+
+  val nino: String = "123456789"
+  val taxYear: Int = 2021
+  val benefitId: String = "a1e8057e-fbbc-47a8-a8b4-78d9f015c934"
+
+  val desUrl = s"/income-tax/income/state-benefits/$nino/${desTaxYearConverter(taxYear)}\\?benefitId=${benefitId}"
+
+  val minStateBenefits = StateBenefitCommonObjectsType(
+    benefitId = benefitId,
+    startDate = "2019-11-13",
+    None,
+    None,
+    None,
+    None,
+    None
+  )
+  val fullStateBenefits: StateBenefitCommonObjectsType = StateBenefitCommonObjectsType(
+    benefitId = benefitId,
+    startDate = "2019-11-13",
+    dateIgnored = Some("2019-04-11T16:22:00Z"),
+    endDate = Some("2020-08-23"),
+    amount = Some(1212.34),
+    submittedOn = Some("2020-09-11T17:23:00Z"),
+    taxPaid = Some(22323.23)
+  )
+
```

Comment:
This is a bit pedantic but if you are changing stuff anyway there is extra line spacing here not required